### DISCUSSION TITLE
[ FF7 ] Game takes ~90 seconds to launch

### DISCUSSION THREAD

**Discussion by malvarenga123** [julianxhokaxhiu/FFNx#302] [Q&A] [answered]
**Describe the bug**
After double-clicking ff7.exe, it takes around 90 seconds for the game to launch.

**To Reproduce**
Clean install of 98 version of FF7.
Update to 1.02.
Copy all original movie files to data/movies.
Extract v1.10.0.0 of FFNx into FF7's directory.
Extract myst6re psf audio pack into FF7's directory.
Run FFNx.reg.
Edit FFNx.toml.
Run ff7.exe
Wait until rundll32.exe closes.
Game will launch.

**Expected behavior**
Game launches imediately.

**Screenshots**
![image](https://user-images.githubusercontent.com/61893389/140406643-8120ca67-1c26-4b23-9d28-dccd8cfcdf4b.png)

![image](https://user-images.githubusercontent.com/61893389/140406880-442b7427-f328-41e2-b137-d4adca733260.png)

**GPU (please complete the following information):**

- AMD Radeon R9 200 series
- Driver Version:  21.09.03.08-210511a-368457C-RadeonSoftware

**Additional context**
As soon as I launch ff7.exe, it will launch rundll32.exe. Both processes barely use any CPU at this point.
Only when rundll32.exe closes by itself will the game actually launch, as seen in the pictures.

Changes in FFNx.toml:
`renderer_backend = 5`
`window_size_x = 1280`
`window_size_y = 960`
`use_external_music = true`
`external_music_resume = false`
`external_music_path = "psf"`
`external_music_ext = "minipsf"`
`he_bios_path = "psf/hebios.bin"`
`show_fps = true`
`show_renderer_backend = true`

FFNx.log: https://pastebin.com/yqWYxgSu

**Comment by julianxhokaxhiu** [comment]
I'm not sure what is going on here but this is the very first time I encounter this report.

I noticed from your logs that you're using the PSF music files, can you please try if the same happens using the config in vanilla mode ( as it comes from the zip file )?

Thank you in advance

**Comment by malvarenga123** [comment]
Disabled external music, but same thing.
Log: https://pastebin.com/y6yzMydH

Vanilla config, same thing.
Log: https://pastebin.com/VJmKsrCa

**Comment by malvarenga123** [comment, marked as answer]
I found out what the problem was. It's related to Windows Game Explorer, which was trying to connect to the internet (the rundll32 thread was calling gameux.dll) to get info for the game. I tried various solutions, but the only one that worked was actually deleting the registry entries preventing the game to be readded to its games list.

The solution is described here: https://web.archive.org/web/20160907130920/http://www.neowin.net/forum/topic/887850-removing-games-explorer/#comment-592577038

**Reply by julianxhokaxhiu** [reply]
Thank you for investigating and for having found the core issue. I usually disable all those overlays ( Steam, Nvidia GeForce Experience, Microsoft Game Bar, etc. ) exactly because they are known to give issues in the end, this is why I never faced the issue myself first. Glad you made it out, enjoy FFNx!
